kgtk/cli/build_faiss.py

```diff
@@ -0,0 +1,111 @@
+"""
+Train and populate a faiss index that can compute nearest neighbors of given embeddings.
+"""
+
+from argparse import Namespace
+from kgtk.cli_argparse import KGTKArgumentParser
+
+
+def parser():
+    return {
+        'help': 'Train and populate a faiss index that can compute nearest neighbors of given embeddings.',
+        'description': 'Train and populate a faiss index that can compute nearest neighbors of given embeddings.'
+    }
+
+
+def add_arguments_extended(parser: KGTKArgumentParser, parsed_shared_args: Namespace):
+    """
+    Parse arguments
+    Args:
+        parser (kgtk.cli_argparse.KGTKArgumentParser)
+    """
+
+    # import modules locally
+    from kgtk.io.kgtkreader import KgtkReader, KgtkReaderOptions, KgtkReaderMode
+    from kgtk.utils.argparsehelpers import optional_bool    # not used yet
+    from kgtk.value.kgtkvalueoptions import KgtkValueOptions
+
+    _expert: bool = parsed_shared_args._expert
+
+    # REQUIRED #
+    # Related to input file
+    parser.add_argument('-i', '--input_file', '--embeddings_file', action='store', dest='embeddings_file',
```

Comment:
There is a `add_input_file` function, please use it
https://github.com/usc-isi-i2/kgtk/blob/master/kgtk/cli/cat.py#L45

Reply:
done